src/components/Onboarding/onboarding-third-step.js

```diff
@@ -66,23 +67,26 @@ class OnboardingThirdStep extends Component {
 		const endWorkAtDate
 			= endWorkAt && new Date(`${currentDate}T${endWorkAt}`);
```

Comment:
```suggestion
			= new Date(`${currentDate}T${endWorkAt}`);
```
This does not work with toString() anymore

Reply:
Same with startWorkAt this needs to be removed